glommio/src/executor/mod.rs

```diff
@@ -566,6 +578,28 @@ impl LocalExecutorBuilder {
         self
     }
 
+    /// Whether to detect stalls in unyielding tasks.
+    /// [`DefaultStallDetectionHandler`] installs a signal handler for
+    /// [`nix::libc::SIGUSR1`], so is disabled by default.
+    /// # Examples
+    ///
+    /// ```
+    /// use glommio::{DefaultStallDetectionHandler, LocalExecutorBuilder};
+    ///
+    /// let local_ex = LocalExecutorBuilder::default()
+    ///     .detect_stalls(Some(Box::new(DefaultStallDetectionHandler {})))
+    ///     .make()
+    ///     .unwrap();
+    /// ```
```

Comment:
Can you use the same doc as in there? https://github.com/DataDog/glommio/pull/499/files#diff-5b724ff10c950f45f86b8b247e7b6000f5f6cba737c7ca8e2ebe1ec5e85bc560R876-R899

Reply:
But they're different? `LocalExecutorBuilder` only takes the stall handler directly. `LocalExecutorPoolBuilder` takes a handler factory function, and its doc uses `on_all_shards` which isn't on `LocalExecutorBuilder`?